### ISSUE TITLE
Terminal is broken after script finishes when running with a Manager in `screen`

### ISSUE THREAD

**Issue by adamnovak** [Rockhopper-Technologies/enlighten#19]
Here's a problem:

1. Grab the example "Advanced" script from [the README](https://github.com/Rockhopper-Technologies/enlighten#advanced) and save as progress.py:

```
import time
import enlighten

manager = enlighten.get_manager()
ticks = manager.counter(total=100, desc='Ticks', unit='ticks')
tocks = manager.counter(total=20, desc='Tocks', unit='tocks')

for num in range(100):
    time.sleep(0.1)  # Simulate work
    print(num)
    ticks.update()
    if not num % 5:
        tocks.update()

manager.stop()
```

2. Start a screen session:

```
$ screen
```

3. Run the script:

```
$ python3 progress.py
```

The script will run through, but something goes wrong during cleanup. The terminal gets into a bad state and you can't see anything you type. It ends up looking something like this:

![image](https://user-images.githubusercontent.com/5062495/82716641-64749c00-9c4d-11ea-85ff-240bd7b732a2.png)

Running `reset` is necessary to fix the terminal.

This may be related to #16, but it's hard to tell. I think this is the root cause of https://github.com/DataBiosphere/toil/issues/3069

**Comment by adamnovak** [comment]
Here's a log from `script`; if you `cat` this inside `screen` the same problem occurs. Outside `screen` it displays fine. Somewhere in the hex dump of this one would be able to see the problem, if one were an expert on terminals and `screen`:

[typescript.txt](https://github.com/Rockhopper-Technologies/enlighten/files/4670814/typescript.txt)

**Comment by avylove** [comment]
The problem seems to be that screen is really picky about the height of the scroll region and will not tolerate a csr value larger than the terminal height. Most terminals don't care and just use the terminal height if the number is bigger.

Why are we giving a value that is bigger? Well, the terminal escape code for csr on most terminals has a %i in it which adds 1 to the values given, except it didn't used to do that in Blessed, but I fixed that last year, so now it does. And it didn't seem to affect other terminals, except now it does for screen, so thanks for reporting it.

I put a fix in and pushed it to master. Please try it out and let me know if it works for you, with and without screen. I still need to test this on Windows, but it's late here, so that will have to wait til tomorrow.

**Comment by avylove** [comment]
I did some more testing and released [1.5.2](https://github.com/Rockhopper-Technologies/enlighten/releases/tag/1.5.2), let me know if this resolves the issue.

I also noticed you have TODO for setting the background color on the progress bar [here](https://github.com/DataBiosphere/toil/blob/1ae5547fe93c73f68e3eb7f5d3dd05aeceb413ab/src/toil/leader.py#L200), so I also included a [documentation update](https://python-enlighten.readthedocs.io/en/stable/examples.html#color) in this release that should help.

Let me know how it works out.

**Comment by avylove** [comment]
Based on comments in DataBiosphere/toil#3069, this seems to be working. Closing. Let me know there are any more issues.
